plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/pinery/PinerySource.java

```diff
@@ -352,6 +352,18 @@ public PinerySource(Path fileName, String instanceName) {
         .collect(Collectors.toSet());
   }
 
+  @ShesmuMethod(
+      name = "$_projects_with_secondary_scheme",
+      description =
+          "Projects marked with the secondary naming scheme from in Pinery defined in {file}.")
+  public Set<String> secondaryProjects() {
+    return projects
+        .get()
+        .filter(SampleProjectDto::isSecondaryNamingSCheme)
```

Comment:
`SCheme` -> `Scheme`?

Reply:
Unfortunately, it is that way in Pinery and we didn't notice, so just rolling with it.

Reply:
:grimacing: good job us